chapter_preface/index_vn.md

```diff
@@ -511,7 +511,19 @@ future editions of this book will be able to provide insights
 in places the present edition cannot.
 -->
 
-*dịch đoạn phía trên*
+Hầu hết các phần của cuốn sách đều bao gồm mã nguồn hoạt động được, bởi vì
+chúng tôi tin rằng trải nghiệm học bằng cách tương tác đóng vai trò quan trọng trong học sâu.
+Hiện tại, một số kĩ năng nhất định chỉ có thể được rèn luyện thông qua việc cứ dùng thử và kiểm tra lỗi,
```

Comment:
```suggestion
Hiện tại, một số kĩ năng nhất định chỉ có thể được rèn luyện thông qua thử nghiệm và nghiên cứu,
```

Ở dưới là các synonyms của "trial and error", mình thấy cụm này phù hợp với ý nghĩa của từ này

https://www.powerthesaurus.org/trial-and-error

Reply:
cái này nên dịch đủ là "thử nghiệm và kiểm tra lỗi"

Reply:
từ nghiên cứu không thể hiện ý nghĩa gốc.